test/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "types": [
+      "@kbn/test/types/expect.js",
```

Comment:
@timroes the expect.js types from npm define a global `expect` variable, which I assume is important for some usages of the library, but is colliding poorly in the Kibana repo where expect.js and jest need to coexist. Since we are somewhat special in this regard and expect.js is effectively _done_, I copied the type definition into `@kbn/test`, changed it to a simple module type, and referenced it from the relevant config files. It won't have the same autoloading benefits of a `@types/` package, but I think it's a suitable solution for the problem. Do you agree?

Reply:
I think that's a good behavior for now, and I currently don't see a reason why it should be bad.